spec/moonshot/commands/build_spec.rb

```diff
@@ -0,0 +1,19 @@
+describe Moonshot::Commands::Build do
```

Comment:
This test looks fine, I was thinking if its possible to put the skip-ci-status test in the test file covering github release if there is one.

Reply:
there is another unit test for github release already merged in previous PR .. this is for build command calls ..  you can find it here
https://github.com/acquia/moonshot/blob/master/spec/moonshot/build_mechanism/github_release_spec.rb#L193

Reply:
okay